lib/Ohmbrewer_RIMS.cpp

```diff
@@ -359,7 +359,11 @@ int Ohmbrewer::RIMS::doUpdate(String &args, Ohmbrewer::Equipment::args_map_t &ar
  */
 std::list<int>* Ohmbrewer::RIMS::whichPins() const {
     // TODO: Implement RIMS::whichPins
+
+    //TODO return array of pins
```

Comment:
You should implement whichPins() as part of this PR... If you fix up the return statement on line 366 (and implement whichPins() properly in the getters it uses), you should be pretty much there.

One unfortunate thing is that whichPins should probably be more like:
```void whichPins(std::list<int>* pinList)```
or
```void whichPins(int* pinList)```
where pinList is a buffer created in the calling function. This is because if you create the pinList within the method itself you'll either get a blank pointer back or (possibly) create a memory leak. See Temperature::toStrC as an example using a character string. It isn't any more complicated to do it which a std::list, and the list would concatenate more easily.